src/sandbox/components/CellEditor.svelte

```diff
@@ -1,23 +1,197 @@
 <script>
+  import { tick, onMount } from "svelte";
+
+  import ProgressiveEval from "../../runtime/engine/progressive_assembly.js";
+  import * as environment from "../../runtime/engine/reserved_words.js";
+  import { getCaretPosition } from "../../utils/highlight";
```

Comment:
extension .js 
It works with our bundler, but maybe not forever. At least this is the convention for now: we don't rely on node resolution algorithm whenever possible